.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+Hi there,
+
+Thank you for opening an issue. Please note that we try to keep the Terraform
+issue tracker reserved for bug reports and feature requests. For general usage
+questions, please see: https://www.terraform.io/community.html.
+
+### Terraform Version
+Run `terraform -v` to show the version. If you are not running the latest
+version of Terraform, please upgrade because your issue may have already been
+fixed.
+
+### Affected Resource(s)
+Please list the resources as a list, for example:
```

Comment:
Maybe also something like: "if the issue appears to affect multiple resources and may be an issue with the core of Terraform, please also mention this".